modules/battenberg/1.1/battenberg.smk

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Ryan Morin
+# Module Author:    Ryan Morin
+# Contributors:     N/A
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+import glob
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["battenberg"]`
+CFG = op.setup_module(
+    name = "battenberg",
+    version = "1.1",
+    subdirectories = ["inputs", "infer_sex","battenberg", "outputs"],
+)
+
+#set variable for prepending to PATH based on config
+SCRIPT_PATH = CFG['inputs']['src_dir']
+#this is used in place of the shell.prefix() because that was not working consistently. This is not ideal. 
+
+#this preserves the variable when using lambda functions
+_battenberg_CFG = CFG
+
+# Define rules to be run locally when using a compute cluster
+# TODO: Replace with actual rules once you change the rule names
+localrules:
+    _install_battenberg,
+    _battenberg_input_bam,
+    _battenberg_output_seg,
+    _battenberg_to_igv_seg,
+    _battenberg_cleanup,
+    _battenberg_all,
+
+
+##### RULES #####
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _battenberg_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam.bai",
+        crai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam.crai"
+    group: "setup_run"
+    run:
+        op.absolute_symlink(input.bam, output.bam)
```

Comment:
ABSOLUTE!

Reply:
Do you want to add assert here, to ensure the correct oncopipe version will be used? The same goes for `in_module` at the output rule.

Reply:
Sure. Do you have an example I can copy?

Reply:
I believe this should work:
```
from packaging import version

try:
    assert version.parse(oncopipe.__version__) >= version.parse('1.0.11')
except AssertionError:
    raise( AssertionError( "Oncopipe version should be equal to or higher than 1.0.11. Please update oncopipe in your environment" ) )
```
It will need packaging added to the environment and can be installed from pip.
I can also think of a way to check the version without the need for additional packages and environment modification.

Reply:
Shouldn't this work? I see that it doesn't but I don't understand why. 
```
>>> print(op.__version__)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'oncopipe' has no attribute '__version__'
```

Reply:
@Lakshay-sethi you probably know the best way to deal with this, right?